imap/squatter.c

```diff
@@ -245,6 +246,13 @@ static int should_index(const char *name)
         goto done;
     }
 
+    // skip CalDAV scheduling Inbox
+    if ((mbtype_isa(mbentry->mbtype) == MBTYPE_CALENDAR) && userid &&
+        !strcmp("Inbox", strarray_nth(mbname_boxes(mbname), 1))) {
```

Comment:
Is `Inbox` hard-coded here to not include http_dav.h for `SCHED_INBOX`?

Reply:
Yeah.  Oversight on my part